src/api/constants/external-apis.ts

```diff
@@ -20,7 +20,15 @@ export const EXTERNAL_API_DEFINITIONS = {
       NO: 'N',
     },
     NO_COVER: 'Off cover',
-    INVALID_COUNTRIES: ['EC Market n/k', 'Non EC Market n/k', 'Non UK', 'Third Country', 'Eastern and Southern African Trade and Development Bank'],
+    INVALID_COUNTRIES: [
```

Comment:
do we need to update any tests for this?

Reply:
Are we sure these aren't being used as well?

Reply:
Good point RE tests, we don't have any tests other than the filtering function and related functions/unit tests.

I have created a ticket to add some E2E test coverage, EMS-3892.